code/index.html

```diff
@@ -7,13 +7,20 @@
     <link rel="stylesheet" href="style.css" />
     <title>Technigo Project Cluedo</title>
   </head>
+
+  <! -- HEADER -->
+  <header>
+  </header>
+
+  <! -- BODY -->
   <body>
     <section class="cards">
       <article class="killer-deck" id="killerCard">
         <p>KILLER</p>
         <p id="killerName"></p>
         <p id="killerAge"></p>
         <p id="killerOccupation"></p>
+        <p id="killerFavoriteWeapon"></p>
         <img id="killerImage" src="" height="100" />
```

Comment:
Add an alt attribute to the img in case the picture isn't visible and for accessibility

Reply:
Very nice Jessica

Reply:
fixed